layouts/partials/head.html

```diff
@@ -1,5 +1,11 @@
-{{- $isBlogPost := eq .Section "blog" }}
-{{- $ogType := cond (.IsHome) "website" "article" }}
+{{/* Adapted from Docsy; changes are to Google Analytics */}}
+{{/* OK to remove once using a modern enough Docsy */}}
```

Comment:
I think it might be worthwhile being more explicit about what "modern enough" means here for a Docsy version – is that possible?

Reply:
Since I wrote this, Docsy have released a modern enough version. I'll revise the file.